src/serializer/modules.js

```diff
@@ -297,6 +297,7 @@ export class Modules {
   moduleTracer: ModuleTracer;
 
   resolveInitializedModules(): void {
+    this.initializedModules.clear();
```

Comment:
Is this needed? Why? I don't think this should happen.

Reply:
This is needed because I can now run this function multiple times to see what modules have been initialized at the end of each additional function serialization/execution. Alternatively I could not report those numbers.